tests/Feature/GetTopsOfTheTopsTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Http\Infrastructure\Clients\DBClient;
+use App\Services\TopsOfTheTopsDataManager;
+use App\Services\TopThreeProvider;
+use App\Services\VideosProvider;
+use DateTime;
+use Mockery;
+use Tests\TestCase;
+
+class GetTopsOfTheTopsTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function getTopsOfTheTops()
```

Comment:
Lo que comentamos en clase, meter esa parte del gets

```suggestion
    public function getsTopsOfTheTops()
```